lib/Textile/API/index.ts

```diff
@@ -238,6 +258,55 @@ class API {
     return result as string
   }
 
+  searchContacts = async (query: ContactQuery, options: QueryOptions, handler: (contact: Contact) => void): Promise<void> => {
```

Comment:
This all looks good in theory. Have you run it? Seems like it should work.

Reply:
yeah, this all works as expected. what was missing (minus the cancelling function)